### ISSUE TITLE
Splash screen settings in 'capacitor.config.json' are ignored

### ISSUE THREAD

**Issue by go-u** [quasarframework/quasar#5611]
**Describe the bug**
We can increase the display time of the splash screen by `capacitor.config.json`. [ (doc link)](https://capacitor.ionicframework.com/docs/apis/splash-screen)  
This is useful when it takes time to prepare the capacitor app.    

However, because quasar forces off splash screen, the setting is ignored.
The relevant part is below.
https://github.com/quasarframework/quasar/blob/ffaa7d5b2eba4e6bdb22a8329a793307cb0472c1/app/templates/entry/app.js#L54-L56

The following settings are ignored by the above quasar's `app.js`

```
// capacitor.config.json
{
  "plugins": {
    "SplashScreen": {
      "launchShowDuration": 5000
    }
  }
}
```
or
```
// capacitor.config.json
{
  "plugins": {
    "SplashScreen": {
      "launchAutoHide": false
    }
  }
}
```

**To Reproduce**
1. Add above settings to capacitor.config.json
2. Run capacitor app
3. Then, you can see `capacitor.config.json` setting is ignored  
(Because Quasar hide splash screen at `mounted()` part )

**Expected behavior**
I think that quasar is better not to operate splash screen.  
It should be left to the capacitor.

By removing below part, we can configure splash screen with `capacitor.config.json`.  
Even if there is no setting, the capacitor automatically hides the splash screen after 3000ms.
Developers who want to reduce the splash screen time can use `SplashScreen.hide ()` separately.

https://github.com/quasarframework/quasar/blob/ffaa7d5b2eba4e6bdb22a8329a793307cb0472c1/app/templates/entry/app.js#L54-L56

**Comment by yusufkandemir** [comment]
@go-u an option for disabling Quasar's `auto-hide the SplashScreen when the app is loaded` feature has been added in https://github.com/quasarframework/quasar/commit/350d8a4998e18931068933f3b0be4a08795d57f8.

@rstoenescu I think this issue can be closed now.

**Comment by snow-drop** [comment]
Good day. In this link, [https://github.com/quasarframework/quasar/commit/350d8a4998e18931068933f3b0be4a08795d57f8](350d8a4), I found a typo at line 26:

`<% if (ctx.mode.capacitor && capacitor.hideSplashcreen !== false) { %>`

I think it should be `capacitor.hideSplashscreen` instead of `capacitor.hideSplashcreen`. I don't know if it has an impact in the function though and I haven't checked if it was already fixed.

**Comment by yusufkandemir** [comment]
@snow-drop you are right, nice catch. As the misspelled `capacitor.hideSplashcreen` is undefined, `capacitor.hideSplashcreen !== false` is always returning true, and the required things get imported. But since the activating logic is correct, it can enable/disable the splash screen correctly. As a result, `SplashScreen` always gets imported even when the plugin is disabled.
https://github.com/quasarframework/quasar/blob/fdffce55dcb7dfee7bc96ab17f4437cd92e3c4f9/app/templates/entry/app.js#L59-L61

Can you please file a PR for this?

**Comment by rstoenescu** [comment]
@snow-drop thanks for catching this! Fix will be available in "@quasar/app" v1.7.2.

**Comment by snow-drop** [comment]
@yusufkandemir @rstoenescu Sure, no problem. I'm glad I could be of help to this project. ;)
